### ISSUE TITLE
get decimal scale and precision through $metadata? Odata Mscrm

### ISSUE THREAD

**Issue by infagbq** [simple-odata-client/Simple.OData.Client#670]
i am not getting decimal scale and precision through $metadata? Odata Mscrm xml

**Comment by NetTecture** [comment]
Yeah. So what is the question? Simple.Odata.Client is a client for odata. Odata specs do not specify metadata with scale and precision. You do not state that you control the server, so the metadata document can only be assumed to be non changing.

So, what is the question. To the one you ask - the answer is "yes, you do not get decimal scale and precision through $metadata".

If you control the metadat document, you can add a vocabulary for this - but it is not standard in odata and will not be understood by any tool.
